tests/debugger/oegdb/enc/contract.c

```diff
@@ -20,7 +20,7 @@ void assert_debugger_binary_contract_enclave_side()
 {
     OE_TEST(TCS_GSBASE_OFFSET == OE_OFFSETOF(sgx_tcs_t, gsbase));
 
-    printf("Debugger contract validated on enclave side.\n");
+    oe_host_printf("Debugger contract validated on enclave side.\n");
```

Comment:
Why should `oe_host_printf` be used instead of `printf`?

Reply:
was trying to minimize the number of imported ocalls (`oe_host_printf` does not need `fcntl.edl`). But it turns out every tests need to import the `fcntl.edl` anyway to support code coverage analysis, I revert the changes.